notebooks/quickstart/data-scientist/02-explore-dataset.ipynb

```diff
@@ -0,0 +1,481 @@
+{
```

Comment:
<p>Break this into two cells.</p><p>In the first cell:</p><pre class="ql-syntax" spellcheck="false"># Let's select the Family dataset
family_age_dataset=domain_client.datasets[0]
</pre><p>In the second cell:</p><pre class="ql-syntax" spellcheck="false"># Let's view the family age dataset
family_age_dataset
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/7045/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>